backend/Users/Users.ts

```diff
@@ -71,3 +65,16 @@ export async function findUser(email, password) {
     return null;
   }
 }
+
+export async function findUserByEmail(email) {
+  const users = await sequelize.models.User.findAll({
+    where: {
+      email: Sequelize.where(
+        Sequelize.fn("LOWER", Sequelize.col("email")),
+        "ILIKE",
```

Comment:
Using `ILIKE` allows for similar emails to match, rather than only allowing exact matches. We should only allow exact matches.

Reply:
Op.eq implemented.